_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -76413,9 +76413,7 @@
 /area/maintenance/starboard)
 "tgQ" = (
 /obj/structure/water_source/puddle,
-/obj/structure/flora/junglebush/large{
-	pixel_y = 0
```

Comment:
this actually wasn't wrong. set it to 1 or something

Reply:
lol